packages/next/build.mjs

```diff
@@ -18,6 +23,11 @@ await Promise.all([
     ],
   }),
   buildEdgeFunctionTemplate(),
+  // The bundled version of source-map looks for the wasm mappings file as a sibling.
+  copyFile(
+    fileURLToPath(import.meta.resolve('source-map/lib/mappings.wasm')),
```

Comment:
`copyFile()` works with URLs:

```suggestion
   import.meta.resolve('source-map/lib/mappings.wasm'),
```

Reply:
It didn't for me. I got an `ENOENT` on the source file.